server/scripts/get-email-list.js

```diff
@@ -0,0 +1,18 @@
+const { dbClient } = require('../db');
+require('dotenv').config({ path: '../.env' });
+(async function exec(){
+    await dbClient.connect();
+    const res = await dbClient.db.query(
+   `SELECT DISTINCT body->> 'emailAddress' AS email 
+    FROM participants p 
+    JOIN participants_status ps 
+        ON  p.id = ps.participant_id AND ps.status != 'hired'
```

Comment:
This is an inner join by default, right? We would want that because we only want the status and participants that are connected to one another by ID and meet the join criteria?